web/src/test/java/no/nav/modiapersonoversikt/rest/persondata/PersondataFletterTest.kt

```diff
@@ -0,0 +1,382 @@
+        folkeregisterpersonstatus: String = "bosatt",
+        sivilstand: HentPersondata.Sivilstand = sivilstandPerson,
+        sikkerhetstiltak: HentPersondata.Sikkerhetstiltak = sikkerhetstiltakData,
+        dodsbo: HentPersondata.KontaktinformasjonForDoedsbo = kontaktinformasjonDodsbo,
+        tilrettelagtKommunikasjon: HentPersondata.TilrettelagtKommunikasjon = tilrettelagtKommunikasjonData,
+        fullmakt: HentPersondata.Fullmakt = fullmaktPerson,
+        telefonnummer: HentPersondata.Telefonnummer = HentPersondata.Telefonnummer("47", "90909090", 1, metadata),
+        vergemaalEllerFremtidsfullmakt: HentPersondata.VergemaalEllerFremtidsfullmakt = vergemal,
+        foreldreansvar: HentPersondata.Foreldreansvar = foreldreansvarData,
+        forelderBarnRelasjon: List<HentPersondata.ForelderBarnRelasjon> = forelderBarnRelasjonData,
+        deltBosted: HentPersondata.DeltBosted = deltBostedData,
+        bosted: List<HentPersondata.Bostedsadresse> = ukjentBosted("Ukjent adresse"),
+        kontaktadresse: HentPersondata.Kontaktadresse = kontaktadresseData
+    ) = HentPersondata.Person(
+        folkeregisteridentifikator = listOf(HentPersondata.Folkeregisteridentifikator(fnr, "I_BRUK", "FNR")),
+        navn = listOf(gittHentPersondataNavn(navn)),
+        kjoenn = listOf(HentPersondata.Kjoenn(kjonn)),
+        foedsel = listOf(HentPersondata.Foedsel(gittDato(fodselsdato))),
+        adressebeskyttelse = listOf(
+            HentPersondata.Adressebeskyttelse(adressebeskyttelse)
+        ),
+        statsborgerskap = listOf(statsborgerskap),
+        doedsfall = listOf(HentPersondata.Doedsfall(gittDato(dodsdato))),
+        folkeregisterpersonstatus = listOf(HentPersondata.Folkeregisterpersonstatus(folkeregisterpersonstatus)),
+        sivilstand = listOf(sivilstand),
+        sikkerhetstiltak = listOf(sikkerhetstiltak),
+        kontaktinformasjonForDoedsbo = listOf(dodsbo),
+        tilrettelagtKommunikasjon = listOf(tilrettelagtKommunikasjon),
+        fullmakt = listOf(fullmakt),
+        telefonnummer = listOf(telefonnummer),
+        vergemaalEllerFremtidsfullmakt = listOf(vergemaalEllerFremtidsfullmakt),
+        foreldreansvar = listOf(foreldreansvar),
+        forelderBarnRelasjon = forelderBarnRelasjon,
+        deltBosted = listOf(deltBosted),
+        bostedsadresse = bosted,
+        kontaktadresse = listOf(kontaktadresse)
+    )
+
+    private fun gittTredjepartsperson(
+        fnr: String = "98765432100",
+        navn: String = "Tredjepart Relasjon",
+        alder: Int? = 20,
+        adressebeskyttelse: Persondata.AdresseBeskyttelse = Persondata.AdresseBeskyttelse.UGRADERT
+    ) = Persondata.TredjepartsPerson(
+        fnr = fnr,
+        navn = listOf(
+            gittNavn(navn)
+        ),
+        fodselsdato = emptyList(),
+        alder = alder,
+        kjonn = emptyList(),
+        adressebeskyttelse = listOf(Persondata.KodeBeskrivelse(adressebeskyttelse, adressebeskyttelse.toString())),
```

Comment:
Mangler å få dette til å mappes korrekt, det er litt annerledes med typer osv siden dette er tredjepersonsmapperen 🤔 Noen innspill?

Reply:
Mulig jeg misforstår. 
Men jeg tenker det er greit å "hardkode" adressebeskyttelse for tredjepartspersoner her. Testingen av selve mappingen til tredjepartspersoner har en egen test-fil, så det trenger vi ikke å forholde oss til så mye her.

Reply:
Good point 🤔 Fair nok det, egentlig 👍🏼  Da lar jeg det stå sånn egentlig, så skal jeg fikse testene.